installer/Migration/V5_6_0/permission/api.yaml

```
@@ -0,0 +1,16 @@
+##########################
+#        Core
+##########################
+
+apiv2_core_validate_uniqueness:
+  description: 'Core - Validate Uniqueness'
+  api: OrangeHRM\Core\Api\Rest\ValidationUniqueAPI
+  module: core
+  allowed:
+    read: true
+    create: false
+    update: false
+    delete: false
+  permissions:
+    - { role: Admin, permission: { read: true, create: false, update: false, delete: false } }
+    - { role: ESS, permission: { read: true, create: false, update: false, delete: false } }
```

```suggestion
    - { role: ESS, permission: { read: false, create: false, update: false, delete: false } }
```
:thinking: seems all places related to Admin screens. Allowing ESS users to check uniqueness will introduce enumeration attack possibility https://www.upguard.com/blog/what-is-an-enumeration-attack

---

removed API from ESS accessible screens and changed permission